test/app.js

```diff
@@ -106,4 +106,11 @@ describe('Smoke test', () => {
 
     should.throws(test);
   });
+
+  it.only('should return account information', async function test() { // eslint-disable-line prefer-arrow-callback
+    this.timeout(10000);
+    const app = createAuthenticatedTestApp();
+    const accInfo = await app.getAccountInfo();
+    console.log('account info: ', accInfo); // eslint-disable-line no-console
```

Comment:
We could as part of the test perform some write operations and verify they change in the `mutations_done`